.github/workflows/ci_cron_daily.yml

```diff
@@ -6,10 +6,7 @@ on:
     - cron: '0 3 * * *'
   pull_request:
     # We also want this workflow triggered if the 'Extra CI' label is added
-    # The first two are subset of the defaults
     types:
-      - synchronize
```

Comment:
Surely you still want syncronised? Otherwise to re-run a job on new commits you are going to have to unlabel and relabel.

Reply:
Hmm... Good point.